init.c

```diff
@@ -113,6 +113,12 @@ static int mount_cgroups()
 {
 	int ret;
 
+	mkdir("/sys/fs/cgroup/pantavisor", 0555);
```

Comment:
we should only do this in v1 mode imo....

Reply:
This results in /proc/PID/cgroup that look like this:

```
1:name=pantavisor:/
0::/lxc/pvr-sdk
```

With this, we can check in ctrl if the PID has "name=pantavisor". This adds an extra layer of security to differentiate between pantavisor and non-pantavisor lxc containers.

Also, it prevents from executing more than one instances of Pantavisor from an early stage.

Reply:
done